### PR TITLE
Added Feature of Restaurant

### PR DESCRIPTION
**Pull request by Ayushi-0903**
Go to search/data_res to test
Analyzed data from NYCOpen Data New York City Restaurant Inspection Results.
Results are retrieved based on the zip code
In the display page (results_res.html), the following information is listed:
Name of the Restraunts: 
Boro: 
ZIP Code: 
Complaint
Violation Description:

### CODE REVIEW

**Comment by Ayushi-0903** [comment]
Hey, I am able to display features now.

File: mainapp/templates/account/login.html [hunk 1]
```diff
@@ -44,7 +46,14 @@ <h1>{% trans "Sign In" %}</h1>
...
         <a class="button secondaryAction" href="{% url 'account_reset_password' %}">{% trans "Forgot Password?" %}</a>
-        <button class="btn btn-primary primaryAction" type="submit">{% trans "Sign In" %}</button>
+        <div class="row">
+        <div class="col-lg">
+          <button class="btn btn-primary primaryAction" type="submit">{% trans "Sign In" %}</button>
+        </div>
+        <div class=""col-lg">
+        <button type="button" class="btn btn-primary" onclick="history.back()">Back</button></a>
+        </div>
+        </div>
```
Comment (ab7289) [inline mainapp/templates/account/login.html]: is there a reason why the `/accounts/login.html` is being updated in your PR?

File: mainapp/templates/account/signup.html [hunk 2]
```diff
@@ -21,7 +23,14 @@ <h1>{% trans "Sign Up" %}</h1>
...
         {% endif %}
-        <button class="btn btn-primary" type="submit">{% trans "Sign Up" %} &raquo;</button>
+        <div class="row">
+        <div class="col-lg">
+          <button class="btn btn-primary" type="submit">{% trans "Sign Up" %} &raquo;</button>
+        </div>
+        <div class=""col-lg">
+          <button type="button" class="btn btn-primary" onclick="history.back()">Back</button></a>
+        </div>
+        </div>
```
Comment (ab7289) [inline mainapp/templates/account/signup.html]: What's the reason for this update? I would think that the restaurant API wouldn't have an affect on the user login template.

**Comment by ab7289** [comment]
I did a request for the 11103 area code.  I'm curious what `Violation Description: nan` is supposed to mean?

image

**Comment by ab7289** [comment]
It appears that the branch has conflicts with develop, could you please update the branch and resolve the conflicts?

File: mainapp/templates/account/login.html (hunk 1 above)
Comment (Ayushi-0903) [reply mainapp/templates/account/login.html]: I didn't update or maybe I updated last time. Now what should I do ?

File: search/templates/search/results_res.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        <p style="color:red">Timeout occurred.</p>
+    {% elif no_matches %}
+        <p style="color:red">No matches were found.</p>
+    {% else %}
+    {% for result in results %}
+        <h5>Apartment Results</h5>
+        <h5>Complaint Results</h5>
+        <ul>
+        <li>Name of the Restraunts: {{ result.dba }}</li>
+            <li>Boro: {{ result.boro }}</li>
```
Comment (mok232) [inline search/templates/search/results_res.html]: nit: can we rename "Boro" to "Borough"?

File: search/templates/search/results_res.html [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+            <li>ZIP Code: {{ result.zipcode }}</li>
+        <li>Complaint</li>
+            <ul>
+            <li>Violation Description: {{ result.violation_description }}</li>
```
Comment (mok232) [inline search/templates/search/results_res.html]: Maybe check if `result.violation_description` is nan?

File: search/templates/search/results_res.html (hunk 3 above)
Comment (Ayushi-0903) [reply search/templates/search/results_res.html]: Fixed

**Comment by alldne** [comment]
https://github.com/gcivil-nyu-org/fall2019-cs-gy-6063-team-five_fries/pull/232

**Pull request closed without merging**